napi.h

```diff
@@ -1773,6 +1773,18 @@ namespace Napi {
       static const napi_node_version* GetNodeVersion(Env env);
   };
 
+#if (NAPI_VERSION > 2)
+  class CleanupHook {
```

Comment:
As @gabrielschulhof reviewed before, class name EnvCleanup would be clear. I will rename it.

Reply:
@nadongguri I don’t think referring to `Env` helps unless you’re a Node.js core developer (or embedder) and are familiar with what an `Environment` is.

I’d not include this in the name.

Reply:
Okay. That makes sense.